pkgs/top-level/all-packages.nix

```diff
@@ -27532,6 +27532,8 @@ in
 
   exodus = callPackage ../applications/blockchains/exodus { };
 
+  faraday = callPackage ../applications/blockchains/faraday.nix { };
```

Comment:
```suggestion
  faraday = callPackage ../applications/blockchains/faraday { };
```

Reply:
this is not how the rest of the file tree appears to work.

Reply:
It shouldn't be like this.

Reply:
fixed by 6a47b6b